src/Microsoft.Azure.SignalR.Management/TaskExtensions.cs

```diff
@@ -28,5 +30,20 @@ public static async Task OrTimeout(this Task task, CancellationToken cancellatio
                 }
             }
         }
+
+        public static async Task<RestApiEndpoint> WithTracingIdAsync(this Task<RestApiEndpoint> task, bool enable = false)
```

Comment:
Given the file name is `TaskExtensions`, this method does not belong to here?

Reply:
I think you can move the method to `RestClient.SendAsync`, just pass one more flag.